photon-core/src/main/java/org/photonvision/common/configuration/ConfigManager.java

```diff
@@ -249,6 +250,14 @@ public Path getImageSavePath() {
         return imgFilePath.toPath();
     }
 
+    public Path getCalibrationImageSavePath(Size frameSize) {
+        var imgFilePath =
+                Path.of(configDirectoryFile.toString(), "calibration", "imgs", frameSize.toString())
```

Comment:
This path needs to include the camera unique name

Reply:
facts...